api/api/openapi.bundle.yaml

```diff
@@ -620,7 +626,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RouterDetails'
+                $ref: '#/components/schemas/RouterVersion'
```

Comment:
To standardise the return values of the pre-existing endpoint with either `true` or `false` as the deploy flag for the `PUT` operation, I've made the operation return `RouterVersion` in both cases. Originally, updating a router returns `RouterDetails`, but since the same endpoint is now used for updating/saving a new router version, it would be odd to return 2 different objects. Furthermore, it'll be nice to return the new `RouterVersion` that's created even during an update, as it'll allow users to immediately retrieve the version number of the latest version (while it's being deployed).